src/ApolloProvider.tsx

```diff
@@ -10,7 +10,9 @@ import {
   Store,
 } from 'redux';
 
-import ApolloClient from 'apollo-client';
+/* tslint:disable:no-unused-variable */
+import ApolloClient, { ApolloStore } from 'apollo-client';
```

Comment:
TS compiler was throwing an error without `ApolloStore` not being in scope when potentially returning an `ApolloStore` in `getChildContext`.